samples/javascript_nodejs/05.multi-turn-prompt/multi-turn-prompt.bot

```diff
@@ -1,10 +1,10 @@
 {
-    "name": "multiturn-prompts-bot",
-    "secretKey": "",
+    "name": "multi-turn-prompt",
+    "padLock": "",
     "services": [
         {
             "type": "endpoint",
-            "name": "multiturn-prompts-bot",
+            "name": "multi-turn-prompt",
```

Comment:
We want to have all local services use the "name": "development" pattern.  in services that have a local and an azure/Luis set of services, this gives us the ability to have local and cloud end points.